packages/app/src/systems/Transaction/services/transaction.tsx

```diff
@@ -1,12 +1,31 @@
 /* eslint-disable no-param-reassign */
-import type { Account } from '@fuel-wallet/types';
-import type { TransactionRequest, WalletUnlocked } from 'fuels';
-import { Provider, TransactionResponse } from 'fuels';
+import type { Account, Asset } from '@fuel-wallet/types';
+import type {
+  BN,
+  TransactionRequest,
+  TransactionRequestLike,
+  WalletLocked,
+  WalletUnlocked,
+} from 'fuels';
+import {
+  Address,
+  bn,
+  hexlify,
+  MAX_GAS_PER_TX,
+  NativeAssetId,
+  ScriptTransactionRequest,
+  transactionRequestify,
+  Wallet,
+  Provider,
+  TransactionResponse,
+} from 'fuels';
 
 import type { Transaction } from '../types';
 import { toJSON } from '../utils';
 
+import { ASSET_MAP, isEth } from '~/systems/Asset';
 import { db, uniqueId } from '~/systems/Core';
+import { provider } from '~/systems/DApp/__mocks__/dapp-provider';
```

Comment:
I think we should not use things from mock file inside a service, would prefer to just create the `provider` here